docs/data-sources/encryption_at_rest_private_endpoints.md

```diff
@@ -0,0 +1,49 @@
+# Data Source: mongodbatlas_encryption_at_rest_private_endpoints
+
+`mongodbatlas_encryption_at_rest_private_endpoints` describes private endpoints of a particular cloud provider used for encryption at rest using customer-managed keys.
+
+## Example Usages
+
+```terraform
+data "mongodbatlas_encryption_at_rest_private_endpoints" "single" {
+  project_id     = var.atlas_project_id
+  cloud_provider = "AZURE"
+}
+
+output "number_of_endpoints" {
+  value = length(data.mongodbatlas_encryption_at_rest_private_endpoints.results)
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cloud_provider` (String) Human-readable label that identifies the cloud provider for the private endpoints to return.
+- `project_id` (String) Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.
+
+**NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
```

Comment:
Currently leaving description coming from API spec, but believe this could be much more reduced and compact to avoid noise. Esstenially only leave `Unique 24-hexadecimal digit string that identifies your project.`. Will see if docs has inputs on this.

Reply:
why not changing it then?

Reply:
Only downside is that we deviate from description defined in the API docs. For this case I believe it is worth it as no valuable information is being provided in the context of terraform.
Followed up by adjusting schema descriptions, will do same in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2533